File: debot/src/main/java/com/tomoima/debot/util/DialogUtil.java
package com.tomoima.debot.util;

import android.app.Activity;
import android.app.AlertDialog;
import android.content.DialogInterface;

import com.tomoima.debot.R;

public class DialogUtil {
    public static void showDialog(final Activity activity, String title, String message){
        AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(activity);
        if(!StringUtil.isBlankOrNull(title)){
            alertDialogBuilder.setTitle(title);
        }
        if(!StringUtil.isBlankOrNull(message)){
            alertDialogBuilder.setMessage(message);
        }
        alertDialogBuilder.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
            @Override
            public void onClick(DialogInterface dialog, int which) {
                dialog.dismiss();
            }
        });
        alertDialogBuilder.setCancelable(true);
        AlertDialog alertDialog = alertDialogBuilder.create();
        alertDialog.show();
    }
}

File: debot/src/main/java/com/tomoima/debot/util/DialogUtil.kt
package com.tomoima.debot.util

import android.app.Activity
import android.app.AlertDialog

object DialogUtil {
    fun showDialog(activity: Activity, title: String?, message: String?) {
        val alertDialogBuilder = AlertDialog.Builder(activity)

        if (!title.isNullOrBlank()) {
            alertDialogBuilder.setTitle(title)
        }
        if (!message.isNullOrBlank()) {
            alertDialogBuilder.setMessage(message)
        }
        alertDialogBuilder.setPositiveButton(android.R.string.ok) { dialog, _ -> dialog.dismiss() }
        alertDialogBuilder.setCancelable(true)
        val alertDialog = alertDialogBuilder.create()
        alertDialog.show()
    }
}
